lib/adapters/MochaAdapter.js

```diff
@@ -0,0 +1,112 @@
+import EventEmitter from 'events'
+import { Test, Suite } from '../Data.js'
+
+export default class MochaAdapter extends EventEmitter {
+  constructor (mocha) {
+    super()
+
+    this.mocha = mocha
+    mocha.reporter((runner) => {
```

Comment:
Is this documented to replace the current reporter? Is there an option to add an additional reporter instead?